demos/common/include/iot_ble_numericComparison.h

```diff
@@ -31,6 +31,8 @@
 #ifndef _AWS_BLE_NUMERIC_COMPARISON_H_
 #define _AWS_BLE_NUMERIC_COMPARISON_H_
 
+#include "queue.h"
```

Comment:
What s this for? "queue.h" should not be needed

Reply:
Its a dependency in the for numeric comparison queue.